### ISSUE TITLE
A11y: Wrap content on all pages in a <main> tag

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#331]
## Describe the task

All content on the page (except skip links) should be contained by landmark tags. This is a best practice.

## Acceptance Criteria

- [ ] Content that is not the header/nav/footer/skip link will be contained in a main tag on all pages

## Additional context
https://dequeuniversity.com/rules/axe/4.8/region?application=AxeChrome

https://dequeuniversity.com/rules/axe/4.8/region?application=AxeChrome

**Comment by bacitracin** [comment]
PR: https://github.com/CodeForPhilly/vacant-lots-proj/pull/353
